exp-player/addon/templates/components/exp-lookit-overview.hbs

```diff
@@ -0,0 +1,40 @@
+<div class="col-md-12">
```

Comment:
Markup nitpick, but don't bootstrap grids usually nest columns inside of rows? (not the other way around)

https://getbootstrap.com/css/#grid-example-basic